clubs/email.py

```diff
@@ -1,32 +1,66 @@
-CREATE_MAIL_SUBJECT = 'Thanks for joining the global movement to teach the web!'
+from django.template.loader import get_template
 
-CREATE_MAIL_BODY = """\
-%(username)s,
 
-Thanks for your interest in teaching the web! We are thrilled that you have taken the first step in your journey to becoming a Mozilla Club Captain.
+def get_full_template_path(relative_path):
+    return 'clubs/email/{rel_path}'.format(rel_path=relative_path)
 
-Once your application is approved, a member of our team will get in touch with you to confirm that your club has been approved.
 
-If you have any questions you can send an email to teachtheweb@mozillafoundation.org, tweet @MozLearn or using hashtag #teachtheweb.
+class EmailTemplateContents:
+    pass
 
-Mozilla Clubs team
-"""
 
-CREATE_MAIL_STAFF_SUBJECT = 'Please review: new club has been submitted'
+class EmailTemplate:
+    def __init__(
+        self,
+        plaintext_subject='',
+        plaintext_body='',
+        html_body=None,
+    ):
+        self.plaintext = EmailTemplateContents()
+        self.plaintext.subject = plaintext_subject
+        self.plaintext.body = plaintext_body
+        self.html = EmailTemplateContents()
+        self.html.body = html_body
 
-CREATE_MAIL_STAFF_BODY = """\
-%(username)s would like to add their Club to the map.
 
-Name: %(club_name)s
-Location: %(club_location)s
-Website: %(club_website)s
-Description: %(club_description)s
-Email: %(email)s
+create_club = EmailTemplate(
+    plaintext_subject=get_template(
+        get_full_template_path('create_club_user/subject.txt')
```

Comment:
I'd advocate making this a function that lets us call `get_full_template_path('create_club', 'user', 'subject')` and have that resolve to the correct thing. There's still a lot of string duplication going on here and below.